vpp-manager/vpp_runner.go

```diff
@@ -126,25 +124,26 @@ func (v *VppRunner) Run(drivers []uplink.UplinkDriver) error {
 	return nil
 }
 
-func (v *VppRunner) configurePunt(tapSwIfIndex uint32, ifState config.LinuxInterfaceState) (err error) {
-	for _, ipFamily := range vpplink.IpFamilies {
-		err = v.vpp.PuntRedirect(types.IpPuntRedirect{
-			RxSwIfIndex: vpplink.InvalidID,
-			Paths: []types.RoutePath{{
-				Table:     common.PuntTableId,
-				SwIfIndex: types.InvalidID,
-			}},
-		}, ipFamily.IsIp6)
-		if err != nil {
-			return errors.Wrapf(err, "Error configuring punt redirect")
-		}
+func (v *VppRunner) configurePunt(tapSwIfIndex uint32, ifState config.LinuxInterfaceState, main bool) (err error) {
+	if main {
+		for _, ipFamily := range vpplink.IpFamilies {
+			err = v.vpp.PuntRedirect(types.IpPuntRedirect{
+				RxSwIfIndex: vpplink.InvalidID,
+				Paths: []types.RoutePath{{
+					Table:     common.PuntTableId,
+					SwIfIndex: types.InvalidID,
+				}},
+			}, ipFamily.IsIp6)
+			if err != nil {
+				return errors.Wrapf(err, "Error configuring punt redirect")
+			}
 
-		err = v.vpp.PuntAllL4(ipFamily.IsIp6)
-		if err != nil {
-			return errors.Wrapf(err, "Error configuring L4 punt")
+			err = v.vpp.PuntAllL4(ipFamily.IsIp6)
+			if err != nil {
+				return errors.Wrapf(err, "Error configuring L4 punt")
+			}
 		}
```

Comment:
We can probably make this into a `configureGlobalPunt()` function and run this only once in the main loop (as it does not depend on the interface)